Makefile

```diff
@@ -3,15 +3,11 @@ export UID=$(shell id -u)
 default: build start create_db populate test stop clean
 
 .PHONY: build
-build:  ## Build containers
+build: create_empty_secret ## Build containers
 	docker-compose build
 
-.PHONY: build_with_version
-build_with_version:
-	docker-compose build --build-arg TRAVIS_PYTHON_VERSION=$(PYTHON_VERSION)
-
```

Comment:
Since the `TRAVIS_PYTHON_VERSION` variable doesn't get used anymore, https://github.com/lucyparsons/OpenOversight/issues/935 , we can get rid of the `build_with_version` command.

Reply:
We still use the variable to set the python version used in the dockerfile. But renaming it to not include "travis" would make sense

Reply:
I see it now! I'll address it.